incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/config/OtlpEndpointAdapter.java

```diff
@@ -24,6 +24,7 @@
 public class OtlpEndpointAdapter implements JsonbAdapter<OtlpEndpointConfig, JsonObject>
 {
     private static final String PROTOCOL_NAME = "protocol";
+    private static final String DEFAULT_PROTOCOL = "http";
```

Comment:
Please rename to `PROTOCOL_DEFAULT`, for consistency with `PROTOCOL_NAME`.

Reply:
fixed